lib/project_copier.rb

```diff
@@ -0,0 +1,30 @@
+class ProjectCopier
+  EXCLUDE_ATTRIBUTES = [ :classifications_count, :launched_row_order, :beta_row_order ].freeze
+  INCLUDE_ATTRIBUTES = [  :project_contents,
+                          :tutorials,
+                          :field_guides,
+                          :pages,
+                          :tags,
+                          :tagged_resources,
+                          :avatar,
+                          :background,
+                          { active_workflows: [ :tutorials, :attached_images, :workflow_contents ] }
+                        ].freeze
+
+  def self.copy(project_id, user_id)
+    project = Project.find(project_id)
```

Comment:
what happens if i can't find the project / user by ID?

Reply:
✔️ Good call, added a rescue to the worker.